lib/pfconfig/namespaces/resource/UriFilters.pm

```diff
@@ -1,44 +1,33 @@
-package pfconfig::namespaces::resource::Profile_Filters;
-
+package pfconfig::namespaces::resource::UriFilters;
 =head1 NAME
 
-pfconfig::namespaces::resource::Profile_Filters
+pfconfig::namespaces::resource::UriFilters
```

Comment:
Ok then mixed/camel case for resources with an underscore after any acronym / abbreviation that is CAPITALIZED.